bitnami/keycloak/templates/NOTES.txt

```diff
@@ -49,7 +49,7 @@ To access Keycloak from outside the cluster execute the following commands:
 
   echo Username: {{ .Values.auth.adminUser }}
 {{- if not .Values.auth.existingSecretPerPassword }}  
-  echo Password: $(kubectl get secret --namespace {{ .Release.Namespace }} {{ printf "%s-env-vars" (include "common.names.fullname" .) }} -o jsonpath="{.data.admin-password}" | base64 --decode)
```

Comment:
Thanks for catching this issue, I think the entire `printf` is a typo.

The correct secretName would be obtained by using:
```
{{ include "common.secrets.name" (dict "existingSecret" .Values.auth.existingSecret "context" $) }}
```

https://github.com/bitnami/charts/blob/c0544b4be0281932d26d1a201e8ddf1d6fa405ea/bitnami/keycloak/templates/secrets.yaml#L1-L7